.github/workflows/test.yml

```diff
@@ -78,7 +78,7 @@ jobs:
         uses: VeryGoodOpenSource/very_good_coverage@v2
         with:
           path: lcov.info
-          min_coverage: 40
+          min_coverage: 39
```

Comment:
Is min coverage going to remain around this level? We can also remove it if we do not prioritize this metric. Not that there's something wrong with this check still being here